scripts/eda_kube.sh

```diff
@@ -228,7 +228,7 @@ case ${CMD} in
   "clean") clean-deployment "${VERSION}";;
   "deploy") deploy "${VERSION}" ;;
   "port-forward-api") port-forward-api 8000 ;;
-  "port-forward-ui") port-forward-ui 8080 ;;
+  "port-forward-ui") port-forward-ui 8443 ;;
```

Comment:
in line 193 we need to change the port from 8080 to 8443. 
even with this change I still got error:
```
(rulebook) billwei@bilwei-mac eda-server % kubectl port-forward svc/eda-ui 8443:8443
Forwarding from 127.0.0.1:8443 -> 443
Forwarding from [::1]:8443 -> 443
Handling connection for 8443
E0921 15:32:03.540365   11077 portforward.go:409] an error occurred forwarding 8443 -> 443: error forwarding port 443 to pod 442050f0ce9cacbb4b5b28d7d39d8e26f453b4c66b18e72d80978a9d260aa216, uid : exit status 1: 2023/09/21 19:32:03 socat[51063] E connect(5, AF=2 127.0.0.1:443, 16): Connection refused
```

Reply:
This error may have to do with https://github.com/kubernetes/kubectl/issues/1169

Reply:
line 193 updated, I can not reproduce the mentioned error.

Reply:
The problem is because minikube is not consuming the eda-ui image built from the project but we use our custom dockerfile/nginx conf so minikube doesn't need to be updated. 
I think we should consume the image provided by ansible-ui team, but that can be changed in a different PR.